README.md

```diff
@@ -1,17 +1,17 @@
 Berserker's Multiworld
 ======================
 
-A Multiworld implementation for the Legend of Zelda: A Link to the Past Randomizer  
-For setup and instructions there's a [Wiki](https://github.com/Berserker66/MultiWorld-Utilities/wiki)  
+A Multiworld implementation for the Legend of Zelda: A Link to the Past Randomizer.  
+For setup and instructions there's a [Wiki](https://github.com/Berserker66/MultiWorld-Utilities/wiki).  
 Downloads can be found at [Releases](https://github.com/Berserker66/MultiWorld-Utilities/releases), including compiled windows binaries.  
 
 Additions/Changes compared to Bonta's V31
 -----------------
 
 Project
- * Available in precompiled form and guided setup for Windows 64Bit on [Releases](https://github.com/Berserker66/MultiWorld-Utilities/releases) page.
- * Compatible with Python 3.7 and 3.8. Forward Checks for Python 4.0 are done.
- * Update modules if they are too old, preventing a crash when trying to connect among potential other issues
+ * Available in precompiled form and guided setup for Windows 64Bit on the [Releases](https://github.com/Berserker66/MultiWorld-Utilities/releases) page
+ * Compatible with Python 3.7 and 3.8. Forward Checks for Python 4.0 are done
+ * Update modules if they are too old. This prevents crashes when trying to connect among potential other issues
```

Comment:
```suggestion
 * Update modules if they are too old to prevent crashes and other possible issues.
```
Two sentences in a bullet point seems like a lot, I figure that condensing them but retaining the intent is good.